modules/gitops-additional-steps-for-disconnected-clusters.adoc

```diff
@@ -0,0 +1,43 @@
+[id="additional-steps-disconnected-clusters_{context}"]
+= Additional steps for disconnected clusters
+
+In a disconnected cluster, Keycloak communicates with OpenShift Oauth Server through proxy. Follow the additional steps to get Keycloak integrated with OpenShift Oauth Login.
+
+.Procedure
```

Comment:
Really great procedure here! Good action verbs, quick and to the point commands, and easily copied code/text!